### PR TITLE
Add Essential Support

### PR DESCRIPTION
**Pull request by ChachyDev**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,33 +1,12 @@
...
 
-apply plugin: 'net.minecraftforge.gradle.forge'
-apply plugin: "com.github.johnrengelman.shadow"
-
-version = "1.8"
-group = "dev.gaminggeek.mojangstatus" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
-archivesBaseName = "MojangStatus"
-
-sourceCompatibility = targetCompatibility = 1.8
+version "1.8"
```
Comment (GamingGeek) [inline build.gradle]: Version should probably be bumped to 2.0

File: src/main/java/dev/gaminggeek/mojangstatus/MojangStatus.java [hunk 2]
```diff
@@ -13,16 +14,13 @@
...
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
-import net.modcore.api.ModCoreAPI;
 
-import static org.koin.core.context.ContextFunctionsKt.startKoin;
-
-@Mod(name = "Mojang Status", modid = MojangStatus.MODID, version = MojangStatus.VERSION)
+@Mod(name = "MojangStatus", modid = MojangStatus.MODID, version = MojangStatus.VERSION)
 public class MojangStatus {
     public static final String MODID = "mojang_status";
     public static final String VERSION = "1.8";
```
Comment (GamingGeek) [inline src/main/java/dev/gaminggeek/mojangstatus/MojangStatus.java]: Version bump here too

File: src/main/java/dev/gaminggeek/mojangstatus/StatusConfig.java [hunk 3]
```diff
@@ -60,7 +60,7 @@
             category = "Global",
             subcategory = "Other"
     )
-    public boolean debug = false;
+    public boolean debug = true;
```
Comment (GamingGeek) [inline src/main/java/dev/gaminggeek/mojangstatus/StatusConfig.java]: Why was this changed to be true?

**Review by GamingGeek — CHANGES_REQUESTED** [review]

File: src/main/java/dev/gaminggeek/mojangstatus/StatusConfig.java (hunk 3 above)
Comment (ChachyDev) [reply src/main/java/dev/gaminggeek/mojangstatus/StatusConfig.java]: Accidental
